### DISCUSSION TITLE
Need help with test results visualization

### DISCUSSION THREAD

**Discussion by himadrics** [TechEmpower/FrameworkBenchmarks#6505] [Q&A]
I'm not very familiar with tech empower benchmarks and I'm having trouble in getting test result visualization when I provide a .json file link on https://www.techempower.com/benchmarks/#section=test. I was trying to run php-ngx-pgsql benchmark and here is the link to the json file that I was trying to visualize - https://who.rocq.inria.fr/Julia.Lawall/results.json.

**Comment by NateBrady23** [comment]
There are no frameworks in the "succeeded" section, which seems to me like you were in `--mode verify` instead of `--mode benchmark`

**Reply by himadrics** [reply]
Thank you for replying. As you pointed out, I was using --mode verify. I tried to run the benchmark again with --mode benchmark and now I'm getting the following error:
JSON.parse: unexpected end of data at line 1 column 1 of the JSON data
Here is the link to my new results.json file - https://pastebin.com/wAJK7WUr
Can you tell me what's wrong with the file and how can I fix it?
Thank you.

**Comment by joanhey** [comment]
It's working correctly.

The json in https://pastebin.com/wAJK7WUr, only test gemini framework in plaintext and json tests.
And visualize only these tests.

https://www.techempower.com/benchmarks/#section=test&shareid=2dc97f2d-74a7-4921-b445-14776d02657e&hw=ph&test=plaintext&a=2
